app/docker/rest/response/handlers.js

```diff
@@ -18,6 +18,9 @@ function isJSON(jsonString) {
 // This handler wrap the JSON objects in an array.
 // Used by the API in: Image push, Image create, Events query.
 export function jsonObjectsToArrayHandler(data) {
+  if (!data) {
```

Comment:
@chiptus looks good, could we add a comment to it to keep track of the symptom improvement

Reply:
I don't think a comment is needed here. it's pretty self-sufficient. if data is undefined, return a default value. Probably fixes a range of symptoms that we don't know about. don't you think?

Reply:
right, code is self sufficient to know what technically it does. What we could provide here is the symptom we solve by helping to hide unwanted error message to user. That could be useful for future maintenance and also other similar situation. Really minor and nice to have i think.